GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -214,6 +214,26 @@ public override string GVFSBinDirectoryName
 
             // Documented here (in the addressing section): https://www.unix.com/man-page/mojave/4/unix/
             public override int MaxPipePathLength => 104;
+
+            public override string UpgradeInstallAdvice
+            {
+                get { return "When ready, run " + this.UpgradeConfirmCommand + " to upgrade."; }
+            }
+
+            public override string UpgradeConfirmCommand
+            {
+                get { return "`sudo gvfs upgrade --confirm`"; }

```

Comment:
Same comment here as in WindowsPlatform.  If this string were moved to a constant in the Shared.cs file, you could avoid duplicating it in `GetUpgradeReminderNotificationImplementation`